### ISSUE TITLE
Quick Edit Option

### ISSUE THREAD

**Issue by robsonfgjunior** [elliotcondon/acf#398]
Wouldn't it be great if we had an option that would allow the custom field to be edited through the Quick Edit view?
When creating the field group there could be a checkbox in each field, something like: 
"Enable quick edit"

I imagine this would be complicated (or even impossible) for the repeater and "flexible content" fields. But all the other regular fields could have this functionality.

Is this somehting you considered yet?

**Comment by elliotcondon** [comment]
Hi @robsonfgjunior 

Thanks for the question,

I have not yet looked at the code required for quick edit, but I'm sure it would be possible. The only issue would be the huge amounts of extra HTML on the page and UI issues for the repeater field.

I'll be sure to look into this in the future.

Thanks
E

**Comment by robsonfgjunior** [comment]
Yes. This kind of feature would be a great addition to the plugin!
 I would leave the repeater field out of this for the same reasons. 

Last week I was working on this project where I had a simple checkbox true/false that determines whether a Post shows up in the featured loop in homepage (get_posts with meta_query argument).
It took about an hour to add the "quick edit" functionality but I had to create the checkbox ui manually, load the values from the custom field and register the "save" functions. It worked this time, but doing it manually for more complex field types (like gallery) would be a pain.

**Comment by robsonfgjunior** [comment]
Maybe I can extend the acf_form_post class to render the field properly inside the Quick Edit panel. I'll let you know if I figure something out :)

**Comment by elliotcondon** [comment]
Hi @robsonfgjunior 

Thanks for the feedback, completely agree.

Be sure to checkout a plugin called Admin Columns Pro - https://www.admincolumns.com/

This will add in ACF fields into the admin columns for quick and easy editing!
